release/test_bom.sh

```diff
@@ -199,4 +220,32 @@ function test_generate_pom_release_bom_third_party_portal {
 	rm release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.third.party-${_ARTIFACT_RC_VERSION}.pom
 }
 
+function _get_github_repository_zip {
+	curl \
+		"https://api.github.com/repos/${1}/${2}/zipball/${3}" \
+		--header "Authorization: token ${4}" \
+		--location \
+		--output repository.zip \
+		--silent
```

Comment:
@lucasmiranda0, can we use `lc_curl` instead? I'm concerned with the lack of error message if any issue occurs during the GitHub API invoke.